extension/src/time_weighted_average/accessors.rs

```diff
@@ -54,3 +55,61 @@ fn time_weight_interpolated_average_accessor<'a>(
         }
     }
 }
+
+pg_type! {
+    #[derive(Debug)]
+    struct TimeWeightInterpolatedIntegralAccessor {
+        start : i64,
+        interval : i64,
+        prev : TimeWeightSummaryData,
+    pad : [u8;3],
```

Comment:
Uff, bit late to do anything about this, but seems like we should have made our pg_type objects aligned so they could be composed without needing padding.